applications/luci-app-p910nd/root/usr/share/rpcd/acl.d/luci-app-p910nd.json

```diff
@@ -2,10 +2,17 @@
 	"luci-app-p910nd": {
 		"description": "Grant UCI access for luci-app-p910nd",
 		"read": {
-			"uci": [ "p910nd" ]
+			"uci": [ "p910nd" ],
+			"file": {
+				"/usr/bin/find -L /dev -maxdepth 3 -type c*": ["exec"],
+				"/usr/libexec/opkg-call list-installed": ["exec"],
+			},
+			"ubus": {
+				"luci": [ "p910nd" ],
+			},
 		},
 		"write": {
-			"uci": [ "p910nd" ]
+			"uci": [ "p910nd" ],
```

Comment:
Please omit the trailing comma as it is not strictly valid JsON and not all parsers will accept it.